src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,20 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  bookItems: [],
+};
+
+const bookSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBook: () => {
+
+    },
+    removeBook: () => {
+
+    },
+  },
+});
+
```

Comment:
Please export `addBook` and `removeBook` as actions
```suggestion
export const { addBook, removeBook } = booksSlice.actions
```